### ISSUE TITLE
Demonstrate the neural network structure of each experimental

### ISSUE THREAD

**Issue by little1d** [SwanHubX/SwanLab#420]
## 🤩 Functional description

Being new to deep learning and even pytorch, I have a hard time understanding the intricate network structure in deep learning. What are the roles of the convolutional, pooling and Softmax layers? How are they connected? A visual representation would improve development and learning efficiency.

**I need a visual structure to help me understand urgently!!!**.

## 👍 What problem does this feature solve

1. Each experiment dashboard displays a different diagram of the network architecture to make it easier to compare experiments.
2. The model building process is easier to understand.
3. Make model fine-tuning process more convenient. 

## 👾 What does the proposed application interface look like?

Maybe swanlab.struct? You can add this code under model define process, take this picture as example:
![image](https://github.com/SwanHubX/SwanLab/assets/115958756/164f4ee5-a604-422d-86b6-6e9fc7082b88)

## 🚑 Any additional information [e.g. screenshots].

![image](https://github.com/SwanHubX/SwanLab/assets/115958756/1e5464a1-df1f-4aaf-a28a-725d30441023)
![image](https://github.com/SwanHubX/SwanLab/assets/115958756/895a5491-9a8a-4ea6-a4a8-a5ac9d9b7d44)

**This is the current work in several related fields, I hope it will be helpful to you.**

https://github.com/HarisIqbal88/PlotNeuralNet

https://github.com/julrog/nn_vis
